assembl/views/__init__.py

```diff
@@ -371,12 +371,25 @@ def process_export_list(ls):
         "get_description": get_description(request),
         "get_landing_page_image": get_landing_page_image(),
         "private_social_sharing": private_social_sharing(),
-        "get_topic": get_topic(request)
+        "get_topic": get_topic(request),
+        "get_discussion_url": get_discussion_url()
     })
 
     return base
 
 
+def get_discussion_url():
+    from ..auth.util import get_current_discussion
+    discussion = get_current_discussion()
+    if discussion:
+        if (len(discussion.get_discussion_urls()) == 2):
```

Comment:
Okay, I know I said that you could use one of these, but I take it back. Remove this check block. Instead, use the `FrontendUrls` object (along with the discussion object) to call the `get_discussion_url`. This will ensure the correct URL, including port number, scheme, V1 vs V2 url, etc.